keyword-relationships.json

```diff
@@ -0,0 +1,118 @@
+{
+  // we need to provide detection context, some keywords can be used in multiple contexts with different buffers being inspected
+  {
+    "keyword": "http",
+    "subkeywords": [
+      {
+        "keyword": "none", // not specifying any sticky buffer or frame etc.
+        "subkeywords": [
+          {
+            "keyword": "http_response_frame",
+            "subkeywords": [
+              {
+                "keyword": "http.response_header",
+                "subkeywords": [
+                  {
+                    "keyword": "http.response_line",
+                    "subkeywords": [
+                      {
+                        "keyword": "http.stat_code",
+                        "subkeywords": []
+                      },
+                      {
+                        "keyword": "http.stat_msg",
+                        "schema-fields": [
+                          {
+                            "$ref": "#/$schema/http.status",
+                            "exact-match": false
```

Comment:
note: this is invalid json, a custom reference would need to be created or possibly have the object like 

```jsonc
{
  "ref": {
     "$ref": "#/$schema/http.status"
  },
  "exact-match": false
}
```